crates/nu-utils/src/sample_config/default_env.nu

```diff
@@ -87,3 +87,6 @@ $env.NU_PLUGIN_DIRS = [
 
 # To add entries to PATH (on Windows you might use Path), you can use the following pattern:
 # $env.PATH = ($env.PATH | split row (char esep) | prepend '/some/path')
+
+# To load additional configuration from a custom file use:
+# source-env ($nu.default-config-dir | path join 'custom.nu')
```

Comment:
This should actually be `source` not `source-env`. So that `alias` commands can work

Reply:
i'm not sure why you're making a PR with source-env and telling us it should be source.